### ISSUE TITLE
Possible function name confusion in gonum/stat/roc_example_test.go

### ISSUE THREAD

**Issue by szaydel** [gonum/gonum#176]
It looks to me like function `func ExampleROC_unweighted()` while called unweighted is using and passing weights to stat.ROC,
```
tpr, fpr := stat.ROC(0, y, classes, weights)
```
while the function `ExampleROC_weighted()` does not pass any weights.
```
tpr, fpr := stat.ROC(0, y, classes, nil)
```
This seems backwards to me. I am happy to correct and do a PR if confirmed that this is indeed a flaw.

**Comment by sbinet** [comment]
yep. thanks for spotting it!

**Comment by szaydel** [comment]
np. Let me if I should do a PR with change, or it is too small and easy enough for someone already looking at it. Cheers.

**Comment by sbinet** [comment]
I am not in front of a real keyboard at the moment, so feel free to send a PR.

**Comment by kortschak** [comment]
Closed by #179.
